.tool-versions

```diff
@@ -1,3 +1,3 @@
 ruby 3.0.0
-nodejs lts-erbium
+nodejs 12.22.12
```

Comment:
So this was necessary for the Github action `marocchino/tool-versions-action` to work. However it is now considered bad practice to set codename versions: https://github.com/asdf-vm/asdf-nodejs#partial-and-codename-versions

Reply:
```
$ asdf nodejs resolve lts-erbium --latest-available
12.22.12
```